plasma_framework/contracts/src/exits/payment/routers/PaymentInFlightExitRouter.sol

```diff
@@ -250,8 +250,8 @@ contract PaymentInFlightExitRouter is
     /**
      * @notice Respond to a non-canonical challenge by providing its position and by proving its correctness
      * @param inFlightTx The RLP-encoded in-flight transaction
-     * @param inFlightTxPos The UTXO position of the in-flight exit. The outputIndex should be set to 0.
-     * @param inFlightTxInclusionProof Inclusion proof for the in-flight tx
+     * @param inFlightTxPos UTXO position of the in-flight exiting transaction, which is a position of 0 indexed output
```

Comment:
I think what's important here is the transaction position, and the output pos in it is irrelevant. How about:
`The position of the in-flight exiting transaction. The output index within the position is unused and should be set to 0`